### DISCUSSION TITLE
Is latest Mpdf is supported for PHP8.2?

### DISCUSSION THREAD

**Discussion by apparaog9** [mpdf/mpdf#1901] [Q&A]
I have downloaded 8.1.6 vesion of library.

My local environment is in PHP 8.2.7 version

I was getting below error. If I fix it in library there are other places errors occured. 

**Fatal error: Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in vendor/mpdf/mpdf/src/Mpdf.php:26057**

Could you please help me to solve these issues.

**Comment by finwe** [comment]
Cannot help you unless you share the code example suffering from this error.
